client/my-sites/plan-features/style.scss

```diff
@@ -240,6 +240,10 @@ $plan-features-header-banner-height: 20px;
 
 .plan-features__footer-buttons {
 	text-align: center;
+
+	&.is-empty {
+		height: 56px;
```

Comment:
While it's nice to have the text above the btns at the same height, I think we don't want to introduce the blank space on mobile devices:

![selection_033](https://cloud.githubusercontent.com/assets/4988512/16518311/75af1c74-3f82-11e6-9cfb-8c9f4e21b356.jpg)